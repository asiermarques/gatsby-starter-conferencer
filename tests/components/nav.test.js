import React from "react"
import renderer from "react-test-renderer"
import Nav from "../../src/components/nav"

describe("Nav", () => {
  it("renders correctly", () => {
    const data = [
      {
        "title": "Test",
        "link": "/test"
      }
    ]
    const tree = renderer.create(<Nav items={data}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})