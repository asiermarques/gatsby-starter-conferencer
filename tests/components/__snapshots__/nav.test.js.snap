// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<ul
  className="nav flex-column"
>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="/test"
      title="Test"
    >
      Test
    </a>
  </li>
</ul>
`;
